### ISSUE TITLE
problem: bootswatch is not taken full advantage of throughout the entire site

### ISSUE THREAD

**Issue by gazhayes** [Blockrazor/blockrazor#577]
Problem: bootswatch isn't used throughout the entire site. It should be taken full advantage of so that changing theme is as easy as switching out a bootswatch style sheet.

ping @MadanBhandari

**Comment by MadanBhandari** [comment]
I am used to write custom HTML, CSS for mostly every components. I will update existing as per bootswatch and will keep in mind in future issues.

**Comment by gazhayes** [comment]
@samuelralak you might want to have a look at this one.

**Comment by samuelralak** [comment]
@gazhayes I propose going through site and make issues of routes where Bootswatch styles are not being used. I have spotted some such as `/add-hashpower`, the page isn't properly styled. I can do this

**Comment by gazhayes** [comment]
You could list them all if you like, or you could just do a commit for each one.

**Comment by AshHimself** [comment]
@samuelralak - If its all the same to you, just post which view or views you are doing and post them here as I am going to start doing a few as well.

**Comment by samuelralak** [comment]
@AshHimself already making a list of this, posting them here in a bit

**Comment by AshHimself** [comment]
Legend.

**Comment by samuelralak** [comment]
@AshHimself @gazhayes 

- [top-nav] most custom css can be eliminated to fully utilize bootswatch navbar theme
- [/add-hashpower] form in add-hashpower page could use some styling; borrow from /addCoin
- [/problem/:_id] input element not styled; add form classes
- [/hashpower] add form classes to input elemets
- [/developers] could use some styling, borrow from bounties or problems
- [/profile] wrap form in card element; borrow from addCoin
- [/currency/:slug] currencyDetail page not responsive - UI breaks on smaller devices

** suggestions **
- tables could be wrapped inside card elemets with white background to make them more visible

**Comment by AshHimself** [comment]
Want to just split this and we can knock it out today @samuelralak ? I am sure we can just do a commit for each against this single ticket.

**Comment by samuelralak** [comment]
Yeah sure @AshHimself. am about to go to bed though so you can go ahead start. let me know what items are done so I can pick up from you when I wake up. if you happen to do them all thats ok too 😄

**Comment by gazhayes** [comment]
@samuelralak your list looks pretty good to me.

> tables could be wrapped inside card elements with white background to make them more visible  

That would probably look much better.

**Comment by AshHimself** [comment]
@samuelralak  - I started these mate 

[/hashpower] add form classes to input elemets
[/developers] could use some styling, borrow from bounties or problems
[/profile] wrap form in card element; borrow from addCoin

**Comment by gazhayes** [comment]
@AshHimself did you push everything? I.e. should @samuelralak leave these routes alone, or can he carry on with them?

**Comment by AshHimself** [comment]
I have not pushed those routes there are just halfish done. He can do all
the others though without risk.

**Comment by gazhayes** [comment]
Don't do the /problems route, I'm going to log an issue for structural changes.
